### PR TITLE
Order page nudges

### PR DESCRIPTION
**Pull request by timuric**
Small design nudges to harmonize the view

**PR intended to be tested with API branch:** 

### Screenshots

Before:
Screenshot 2023-05-19 at 09 28 12
After:
Screenshot 2023-05-19 at 09 27 56

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

File: src/orders/components/OrderTransaction/components/CardTitle/CardTitle.tsx [hunk 1]
```diff
@@ -45,8 +46,15 @@ export const CardTitle: React.FC<CardTitleProps> = ({
...
     authorizedAmount,
+    type,
   } = transaction;
 
+  // Capitalize transaction type
+  const title = maybe(
+    () => type[0].toUpperCase() + type.slice(1).toLowerCase(),
+    "Transaction",
+  );
+
```
Comment (witoszekdev) [inline src/orders/components/OrderTransaction/components/CardTitle/CardTitle.tsx:57]:
```suggestion
  const title = capitalize(type || "Transaction")
```

File: src/orders/components/OrderTransaction/components/CardTitle/CardTitle.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import {
   TransactionActionEnum,
   TransactionItemFragment,
 } from "@dashboard/graphql";
+import { maybe } from "@dashboard/misc";
```
Comment (witoszekdev) [inline src/orders/components/OrderTransaction/components/CardTitle/CardTitle.tsx:6]:
```suggestion
import { capitalize } from "@dashboard/misc";
```

File: src/orders/components/OrderTransaction/components/CardTitle/MoneyDisplay.tsx [hunk 3]
```diff
@@ -18,16 +17,12 @@ export const MoneyDisplay = ({ label, money }: MoneyDisplayProps) => {
...
-        <span className={classes.amount}>{amount}</span>
-      </Typography>
-    </dl>
+    <div className={classes.wrapper}>
+      <div>{label}</div>
+      <div>
+        <span>{money.currency}&nbsp;</span>
+        <span>{amount}</span>
+      </div>
+    </div>
```
Comment (witoszekdev) [inline src/orders/components/OrderTransaction/components/CardTitle/MoneyDisplay.tsx:26]: Why are we removing this semantic element? 🤔

File: src/orders/components/OrderTransaction/components/TransactionEvents/components/EventStatus.tsx [hunk 4]
```diff
@@ -16,30 +16,41 @@ export const EventStatus: React.FC<EventStatusProps> = ({ status }) => {
       return (
         <Pill
           color="warning"
+          size={"small"}
```
Comment (witoszekdev) [inline src/orders/components/OrderTransaction/components/TransactionEvents/components/EventStatus.tsx:19]: If we don't that Prettier linting will fail (I'm not sure if it's done right now in dashboard's CI 😅)

```suggestion
          size="small"
```

File: src/orders/components/OrderTransaction/components/TransactionEvents/components/EventType.tsx [hunk 5]
```diff
@@ -26,10 +26,13 @@ export const EventType = ({ type, message }: EventTypeProps) => {
...
   const mapEventToMessage = transactionEventTypeMap[type];
 
-  const displayType = mapEventToMessage
+  let displayType = mapEventToMessage
     ? intl.formatMessage(mapEventToMessage)
     : message || type;
 
+  // Capitalise first letter
+  displayType = displayType.charAt(0).toUpperCase() + displayType.slice(1);
+
```
Comment (witoszekdev) [inline src/orders/components/OrderTransaction/components/TransactionEvents/components/EventType.tsx:35]:
```suggestion
  const displayType = capitalize(
    mapEventToMessage
      ? intl.formatMessage(mapEventToMessage)
      : message || type
    );
```

**Review by witoszekdev — DISMISSED** [review]

**Review by poulch — APPROVED** [review]

**Comment by wojteknowacki** [comment]
Please verify checks before placing ticket in QA, there are jest and translation checks failing

**Comment by poulch** [comment]
@timuric please fix ts and lint checks, also run extract messages.

**Review by poulch — DISMISSED** [review]

**Review by Droniu — DISMISSED** [review]

**Review by wojteknowacki — DISMISSED** [review]

**Review by Droniu — APPROVED** [review]

**Pull request merged**